2sem/common/stack.h

```diff
@@ -0,0 +1,97 @@
+#pragma once
+
+#include <stdexcept>
+
+using namespace std;
+
+/// <summary>
+/// Ниже находится описание класса "Stack", который является системой хранения данных типа "LIFO"
+template <typename T>
+class Stack
+{
+private:
+	struct Node
+	{
+		T data;
+		Node* next;
+
+		//создан конструктор структуры Node 
+		Node(const T& value) : data(value), next(nullptr) {} 
+	};
+
+	int counter;
+	Node* top;
+
+public:
+	/// <summary>
+	/// Конструктор для структуры Stack
+	Stack() : counter(0), top(nullptr) {}
+	/// <summary>
```

Comment:
Разделители между функциональными членами класса
```suggestion

	/// <summary>
```

Reply:
Здесь и далее